include/deal.II/multigrid/multigrid.h

```diff
@@ -74,6 +74,60 @@ class Multigrid : public Subscriptor
     f_cycle
   };
 
+  /**
+   * A structure that has boost::signal objects for a number of actions that
+   * happen for the typical usage of this class.
+   *
+   * For documentation on signals, see
+   * http://www.boost.org/doc/libs/release/libs/signals2 .
+   */
+  struct Signals
+  {
+    /**
+     * This signal is triggered before and after the call to
+     * MGTransfer::copy_to_mg which transfers given vector to all levels.
+     */
+    boost::signals2::signal<void (bool start)> transfer_to_mg;
+
+    /**
+     * This signal is triggered before and after the call to
+     * MGTransfer::copy_from_mg which transfers a multi-level vector
+     * to a normal vector.
+     */
+    boost::signals2::signal<void (bool start)> transfer_to_global;
+
+    /**
+     * This signal is triggered before and after the call to the coarse solver.
+     */
+    boost::signals2::signal<void (const bool start, const unsigned int level)> coarse_solve;
+
+    /**
+     * This signal is triggered before and after the call to
+     * MGTransfer::restrict_and_add() which restricts a vector from @p level
+     * to the next coarser one (@p level - 1).
+     */
+    boost::signals2::signal<void (const bool start, const unsigned int level)> transfer_down_to;
+
+    /**
+     * This signal is triggered before and after the call to
+     * MGTransfer::prolongate() which prolongates a vector to @p level
```

Comment:
We'll need a native speaker of English here -- @drwells, @jppelteret!

My recollection is that we accidentally used the word "prolongate" in the multigrid classes (and the finite element classes) because we didn't know better and that the correct choice would have been "prolong". Searching the internet, you can find the former defined in terms of the latter, so both seem to work. Either way, we can't change the name of the function, but we can change the second occurrence of the word on this line :-)

Reply:
As you said, it would seem that it shouldn't matter since they're both dictionary-accepted verbs. I do find "prolong" easier on the ear in this situation though. Because of this, and to add diversity to the description, maybe I'd lean towards @bangerth's suggestion of writing it as
> ... MGTransfer::prolongate() which prolongs a vector to...

After all, variety is the spice of life :-)

What do you think @drwells ?